assets/js/favorites.js

```diff
@@ -0,0 +1,46 @@
+$("#favorite-star").click(function () {
+    let word = $(this).data('word');
+    if (word) {
+        $(this).hasClass('is-favorite') ? removeWord(word) : addWord(word);
+    }
+    $(this).toggleClass('is-favorite');
+});
+
+$(".btn-remove").click(function (e) {
```

Comment:
dont use bootstrap class for any global actions to prevent misunderstandings in the future - add your own class

Reply:
Its my own class ☺